mzLib.sln

```diff
@@ -61,11 +61,13 @@ Global
 		{D7C69058-1FE4-49BD-B994-B08C1DFB21A4}.Debug|Any CPU.ActiveCfg = Debug|x64
 		{D7C69058-1FE4-49BD-B994-B08C1DFB21A4}.Debug|Any CPU.Build.0 = Debug|x64
 		{D7C69058-1FE4-49BD-B994-B08C1DFB21A4}.Release|Any CPU.ActiveCfg = Release|x64
+		{D7C69058-1FE4-49BD-B994-B08C1DFB21A4}.Release|Any CPU.Build.0 = Release|x64
```

Comment:
I think all the changes in this file should be undone

Reply:
The build order wasn't working for me in "Release" like it was in "TestAndRelease" and "Debug" -- some things just weren't getting built. These changes come from checking all of the projects in editing the "Release" Build Order.